package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@artsy/gemup",
-  "version": "0.0.3",
+  "version": "0.1.0",
```

Comment:
Since there a few other changes that will be included in this release, I would recommend splitting this version bump into a separate PR once this has merged.